pkgs/tools/package-management/nix/default.nix

```diff
@@ -57,6 +57,7 @@ common =
         ++ lib.optional (stdenv.isLinux || stdenv.isDarwin) libsodium
         ++ lib.optionals is24 [ libarchive gmock lowdown ]
         ++ lib.optional (is24 && stdenv.isx86_64) libcpuid
+        ++ lib.optionals (is24 && stdenv.isLinux) [ util-linuxMinimal ]
```

Comment:
If you feel like rebasing again, that would be a case for `lib.optional` :)

Reply:
This was actually intentional: https://github.com/NixOS/nixpkgs/pull/116274#discussion_r593834474

Reply:
Fair enough.